File: reactor-java/src/main/java/com/github/mostroverkhov/r2/java/RequesterBuilder.java
package com.github.mostroverkhov.r2.java;

import com.github.mostroverkhov.r2.core.CoreRequesterBuilder;
import com.github.mostroverkhov.r2.java.adapters.RequesterAdapter;
import io.rsocket.RSocket;

public class RequesterBuilder extends CoreRequesterBuilder {
    public RequesterBuilder(RSocket rSocket) {
        adapter(new RequesterAdapter(rSocket));
    }
}

File: rxjava-kotlin/src/main/java/com/github/mostroverkhov/r2/rxjava/RequesterBuilder.kt
package com.github.mostroverkhov.r2.rxjava

import com.github.mostroverkhov.r2.rxjava.adapters.RequesterAdapter
import com.github.mostroverkhov.r2.core.CoreRequesterBuilder
import io.rsocket.kotlin.RSocket

internal class RequesterBuilder(rSocket: RSocket) : CoreRequesterBuilder() {
    init {
        adapter(RequesterAdapter(rSocket))
    }
}